src/util/rsa.go

```diff
@@ -0,0 +1,32 @@
+package util
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/base64"
+)
+
+func GenerateRsa() (key *rsa.PrivateKey) {
+	// Should never error
+	key, _ = rsa.GenerateKey(rand.Reader, 4096)
```

Comment:
when there's something that we cannot permit to error, i strongly believe that this should be an 

```
if err != nil {
panic(err)
}
```

instead of failing silently, even if we expect it to never happen. maybe it's entropy starved and crypto/rand dnot work idk

Reply:
True, wasn't sure how to approach not returning the error but also not handling it tbh